### ISSUE TITLE
We need a better description of the OWID-Grapher on OWID

### ISSUE THREAD

**Issue by maxroser** [owid/owid-grapher#1262]
We have this description here [https://ourworldindata.org/owid-grapher](https://ourworldindata.org/owid-grapher)

It would be good to update this page.
– It doesn’t mention Explorers at all.
– Lars rightly pointed out that "it describes grapher as being more reusable by others than it really is."

This is not a big project. Maybe much of this could be done in one or two days.

It could be a part of the project that Marcel was working on which has the goal to increase the contribution to our open-source tools.

**Comment by maxroser** [comment]
@danyx23 @marcelgerber (cc @larsyencken) Would it be possible to bring this issue into the current Grapher redesign work? I'm tagging the two of you since you know the Grapher so well, but feel free of course to delegate to whoever has an interest and capacity to do it.

We have an article about our Grapher https://ourworldindata.org/owid-grapher but it isn't great and now very out-of-date.

Even a relatively small effort — two people spending two hours or so — could improve the status quo quite a bit. It'd be nice for all the engineering colleagues if this big part of our work is a bit better presented as in the status quo.

**Comment by danyx23** [comment]
Yeah agreed. I thought we'll just let this issue be closed and do this when we migrate the about pages but it's probably good to keep this open. I think we'll move some of the about pages over to gdocs in the coming weeks and then I'll rewrite the grapher page.

**Comment by marcelgerber** [comment]
We now have a somewhat decent description here: https://ourworldindata.org/faqs#what-software-do-you-use-for-your-visualizations-and-can-i-use-it

Closing.
